libbeat/cmd/configtest.go

```diff
@@ -15,6 +16,7 @@ func genConfigTestCmd(name, version string, beatCreator beat.Creator) *cobra.Com
 		Run: func(cmd *cobra.Command, args []string) {
 			b, err := beat.New(name, version)
 			if err != nil {
+				fmt.Fprintf(os.Stderr, "Error initializing beat: %s\n", err)
```

Comment:
What's your exact concern? I tried to do it just the other way around, strip all log metadata as this is command line

It would be possible though